src/obi_auth/storage.py

```diff
@@ -0,0 +1,30 @@
+"""Storage module."""
+
+from dataclasses import dataclass
+from pathlib import Path
+
+from obi_auth.typedef import TokenInfo
+
+
+@dataclass
+class Storage:
+    """Storage class."""
+
+    file_path: Path
+
+    def write(self, data: TokenInfo):
+        """Write token info to file."""
+        self.file_path.write_text(data.model_dump_json())
```

Comment:
The permissions of the file should be set to 0600, ideally even before writing the content so that it's never readable.

Reply:
Done.